acquire/outputs/zip.py

```diff
@@ -0,0 +1,100 @@
+import io
+import pyzipper
```

Comment:
Any specific reason for not using `zipfile` (https://docs.python.org/3/library/zipfile.html)?

Reply:
Agreed, better use the standard library to avoid unnecessary dependencies. Since this is quite a fundamental issue with this PR I say further reviewing at this point is premature (because changing the library might have a big impact on the overall code).